FAIMS/FAIMS3
# Update end to end testing to pass with the latest alpha version

1. There's a weird bug in appium where it's not picking up the JSON changes for checkbox and radio field, although it's visible through the JSON on the app. So we have to skip checking these 2 fields in the JSON.

2. Workaround for FAIMS3-263: wait for a few seconds to let the records load on "AstroSky" page before opening a new observation form.

3. Update email address according to Brian's UAT spec.

4. New tests for: Doable Task 2.2 - Incomplete/Draft Observations
Tests passed: 
4.1 Android: https://app-automate.browserstack.com/dashboard/v2/builds/da5769c5e33c09360b5337ef71f5cda36b6e0791/sessions/8c416b214d7d27d6492ec92e1dfa91f05b094e90
4.2 Chrome: https://automate.browserstack.com/dashboard/v2/builds/cd350302d64dd9a269c542c7138e291d81016fdf/sessions/8289896e9d3b0bcf80a2be5e7ff1743f6eb7d30b)

5. Enter some values in Android integer field with numeric keypad: https://faimsproject.atlassian.net/browse/FAIMS3-221
6. Update tests with: Doable Task 2.3 - GPS and Taking a Point
(after pressing 'Take Point' and 'Action' buttons, return to projects page and reload the page to check that the values have been updated)
